services/swarm/config.go

```diff
@@ -0,0 +1,70 @@
+package swarm
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"io/ioutil"
+
+	"github.com/influxdata/kapacitor/services/swarm/client"
+	"github.com/pkg/errors"
+)
+
+type Config struct {
+	Enabled            bool     `toml:"enabled" override:"enabled"`
+	APIServers         []string `toml:"api-servers" override:"api-servers"`
+	CAFile             string   `toml:"ca-file" override:"ca-file"`
+	CertFile           string   `toml:"cert-file" override:"cert-file"`
+	KeyFile            string   `toml:"key-file" override:"key-file"`
+	APIVersion         string   `toml:"api-version" override:"api-version"`
+	InsecureSkipVerify bool     `toml:"insecure-skip-verify" override:"insecure-skip-verify"`
+}
+
+func NewConfig() Config {
+	return Config{}
+}
+
+func (c Config) Validate() error {
```

Comment:
I didn't see this function being called in `server/config.go` can you double check?